### DISCUSSION TITLE
BluetoothA2DPSink.h not working with ledcSetup() / ledcWriteNote() ?...

### DISCUSSION THREAD

**Discussion by Knottis** [pschatzmann/ESP32-A2DP#352] [General]

**Comment by pschatzmann** [comment]
Why don't you just analyse the stacktrace?

**Reply by Knottis** [reply]
I am only a **_beginner/amateur_** who does not understand such errors as this...
So I apologize if this is elementary for you guys.

But how do I solve the problem, so that this program works?

_Thank you for your understanding!.._

**Comment by pschatzmann** [comment]
https://github.com/me-no-dev/EspExceptionDecoder

**Reply by Knottis** [reply]
### _Wow... :-0 This is far too complicated for me!. as a beginner/amateur!.._ 

So to put it simply, A2DP is not **compatible** with other Audio/PWM- functions used at the same time on the ESP32-Arduino.

_Thanks for the clarification on this._

**Regards, Knottis**
(Sorry for my bad English)

**Comment by pschatzmann** [comment]
Where did you find this information. I tried your sketch and I don't get any error.
Are you sure that you use a recent version of the ESP32 Arduino implementation ?
The actual version is 2.0.6.

**Reply by Knottis** [reply]
**I use the last version of Arduino IDE**

Version: 2.0.3
Date: 2022-12-05T09:30:25.331Z
CLI Version: 0.29.0 [76251df9]
Copyright © 2023 Arduino SA

**https://www.arduino.cc/en/software**

**and for ESP32**
Arduino core for the ESP32, ESP32-S2, ESP32-S3 and ESP32-C3

_platform.txt_
name=ESP32 Arduino
version=2.0.6

**https://github.com/espressif/arduino-esp32**

**Reply by pschatzmann** [reply]
Strange...

**Comment by Knottis** [comment]
### THE ERROR IS RESOLVED 👍 

**⚙️Found the error, which is that my ESP32 card is _broken/defective_ ♻️.** 

The new one works perfectly.

_😀Thanks for the for quick help._

**Regards, Knottis**
